x-pack/plugins/timelines/public/components/t_grid/helpers.test.tsx

```diff
@@ -518,6 +518,101 @@ describe('Combined Queries', () => {
     );
   });
 
+  test('Disabled Data Provider and kQuery', () => {
```

Comment:
kQuery or `kqlQuery`?

Reply:
It is kqlQuery. Will correct it. still getting acquainted with these terms, sorry :)

Reply:
@michaelolo24 fixed. 🪛